src/views/Simulation/Simulation.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+import React, { useState } from 'react';
+import { Box, Grid } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import { InspectorDrawer, Kpis, SceneContainer, TopBar } from './components';
+
+const useStyles = makeStyles((theme) => ({
+  viewContainer: {
+    height: '100%',
+    width: '100%',
+  },
+}));
+
+const Simulation = () => {
+  const classes = useStyles();
+
+  const [isInspectorDrawerOpen, setIsInspectorDrawerOpen] = useState(false);
+  // const openInspectorDrawer = () => setIsInspectorDrawerOpen(true);
+  const closeInspectorDrawer = () => setIsInspectorDrawerOpen(false);
+  const toggleInspectorDrawer = (previousValue) => setIsInspectorDrawerOpen(previousValue);
+
+  return (
+    <Box sx={{ flexGrow: 1 }} className={classes.viewContainer}>
```

Comment:
Shouldn't we avoid to externalise style css and include style directly on the component tag ?